website_sale_product_brand/views/product_brand.xml

```diff
@@ -58,16 +58,11 @@
     </template>
 
     <!-- Menu in website for Brand -->
-    <record id="menu_website_sale_shop" model="website.menu">
-        <field name="name">Shop All</field>
-        <field name="url">/shop</field>
-        <field name="parent_id" ref="website_sale.menu_shop" />
-    </record>
-
     <record id="menu_website_sale_brand" model="website.menu">
         <field name="name">Shop by Brand</field>
         <field name="url">/page/product_brands</field>
-        <field name="parent_id" ref="website_sale.menu_shop" />
+        <field name="sequence" type="int">25</field>
+        <field name="parent_id" ref="website_sale.main_menu" />
```

Comment:
external id seems to be wrong, website.main_menu
https://github.com/odoo/odoo/blob/9.0/addons/website/data/data.xml#L12

Runbot fails due to this
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3155288-122-6df6af/openerp/addons/base/ir/ir_model.py", line 974, in xmlid_lookup
    raise ValueError('External ID not found in the system: %s' % (xmlid))
ParseError: "External ID not found in the system: website_sale.main_menu" while parsing /srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3155288-122-6df6af/openerp/addons/website_sale_product_brand/views/product_brand.xml:61, near
<record id="menu_website_sale_brand" model="website.menu">
        <field name="name">Shop by Brand</field>
        <field name="url">/page/product_brands</field>
        <field name="sequence" type="int">25</field>
        <field name="parent_id" ref="website_sale.main_menu"/>
    </record>

For more info, check the runbot log